components/script/dom/readablestream.rs

```diff
@@ -303,18 +303,18 @@ impl ReadableStream {
     /// Call into the pull steps of the controller,
     /// as part of
     /// <https://streams.spec.whatwg.org/#readable-stream-default-reader-read>
-    pub fn perform_pull_steps(&self, read_request: ReadRequest) {
+    pub fn perform_pull_steps(&self, read_request: &ReadRequest, can_gc: CanGc) {
```

Comment:
Note that I changed to passing a reference to please the rooting lint, so an alternative to avoid the clone at the end of the various calls is to add the allow tag.